lib/usbmux/index.js

```diff
@@ -88,7 +90,17 @@ class Usbmux extends BaseServiceSocket {
    * @returns {?Object}
    */
   async readPairRecord (udid, timeout = 5000) {
-    const { tag, receivePromise } = this._receivePlistPromise(timeout);
+    const {tag, receivePromise} = this._receivePlistPromise(timeout, (data) => {
+      if (data.payload.PairRecordData) {
+        try {
+          return plist.parsePlist(data.payload.PairRecordData);
+        } catch (err) {
+          throw new Error(`Unexpected data: ${JSON.stringify(data)}`);
```

Comment:
I'd rather `log.debug` the actual error here

Reply:
It must be really strange that parsing fails. It can stay as error